pkg/storaged/utils.js

```diff
@@ -407,11 +407,16 @@ export function get_available_spaces(client) {
             return !!client.legacy_vdo_overlay.find_by_backing_block(block);
         }
 
+        function is_swap() {
+            return !!block && client.blocks_swap[path];
```

Comment:
Can block ever be null here?

Reply:
The rest of the checks explicitly call `block.IdType` which would error out if it was `null` so it seems not.